src/abstraction-map.c

```diff
@@ -32,13 +32,17 @@
 #include "abstraction-map.h"
 
 const struct cgroup_abstraction_map cgroup_v1_to_v2_map[] = {
+	/* cpu controller */
 	{cgroup_convert_int, "cpu.shares", (void *)1024, "cpu.weight", (void *)100},
+	{cgroup_convert_passthrough, "cpu.stat", NULL, "cpu.stat", NULL},
 };
 const int cgroup_v1_to_v2_map_sz = sizeof(cgroup_v1_to_v2_map) /
 				   sizeof(cgroup_v1_to_v2_map[0]);
 
 const struct cgroup_abstraction_map cgroup_v2_to_v1_map[] = {
+	/* cpu controller */
 	{cgroup_convert_int, "cpu.weight", (void *)100, "cpu.shares", (void *)1024},
+	{cgroup_convert_passthrough, "cpu.stat", NULL, "cpu.stat", NULL},
```

Comment:
`cpu.stat` is a little special, as its values are in the `multiline`.  This needs to be handled while passing the value of the setting to `cgroup_covnvert_passthrough` hinting it to write into `cgroup_contoller->multiline_value` and not `cgroup_controller->value` or something like that.

Reply:
The contents of `cpu.stat` aren't a 1-to-1 mapping from cgroup v1 to v2.  Thus, even passing through the multi-line data still wouldn't be correct.  I'm thinking about marking `cpu.stat` as unmappable for now.

We can then revisit it in a smaller patchset :).  There's also `cgroup_read_stats()` which needs to be made smarter for v1/v2.

Thoughts?

Reply:
Agree, for now, it can be marked as un-mappable.